File: src/main/java/coza/trojanc/stonewriter/template/process/fields/ProcessedLine.java
package coza.trojanc.stonewriter.processor.fields;

import java.util.ArrayList;
import java.util.List;

/**
 * Created by Charl-PC on 2016-10-11.
 */
public class ProcessedLine  implements ProcessedLineItem{

	private List<ProcessedText> lineItems = new ArrayList<>();

	public List<ProcessedText> getLineItems() {
		return lineItems;
	}
}

File: receipt-shared/src/main/kotlin/coza/trojanc/receipt/template/process/fields/ProcessedLine.kt
package coza.trojanc.receipt.template.process.fields

/**
 * @author Charl Thiem
 */
class ProcessedLine : ProcessedLineItem {

    private val lineItems: MutableList<ProcessedText> = ArrayList<ProcessedText>()

    fun getLineItems(): List<ProcessedText> {
        return lineItems
    }

    fun addProcessedText(processedText: ProcessedText){
        this.lineItems.add(processedText);
    }
}
